ray-operator/config/samples/ray_v1alpha1_rayservice.yaml

```diff
@@ -9,38 +9,39 @@ metadata:
 spec:
   serviceUnhealthySecondThreshold: 300 # Config for the health check threshold for service. Default value is 60.
   deploymentUnhealthySecondThreshold: 300 # Config for the health check threshold for deployments. Default value is 60.
-  serveConfig:
-    importPath: fruit.deployment_graph
-    runtimeEnv: |
+  serveConfigType: SINGLE_APP
```

Comment:
Do we still need the SINGLE_APP example if our intention is to de-emphasize it?

Reply:
Since the config type still defaults to single app, I think it'd be good to still keep the example. I have changed the default `ray_v1alpha1_rayservice.yaml` file to use multi app, and renamed the single app example to `ray_v1alpha1_rayservice-single-app.yaml` to encourage users to use multi app.

Reply:
Synced offline. We are removing `ray_v1alpha1_rayservice-single-app.yaml`